doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,258 @@
+"""
+============================================
+Restore spotted cornea image with inpainting
+============================================
+
+Optical coherence tomography (OCT) is a non-invasive imaging technique used by
+ophthalmologists to take pictures of the back of a patient's eye [1]_.
+When performing OCT,
+dust may stick to the reference mirror of the equipment, causing dark spots to
+appear on the images. The problem is that these dirt spots cover areas of
```

Comment:
```suggestion
appear on the images. The problem is that these dust spots cover areas of
```
Replace "dirt" with "dust" to remain consistent through this section.